build.sbt

```diff
@@ -11,6 +11,8 @@ ThisBuild / crossScalaVersions := Seq(Scala212, Scala213, Scala3)
 
 ThisBuild / scalaVersion := Scala213
 
+lazy val previousMouseVersion = "1.0.7"
+
```

Comment:
One of the nice things about current CI Release process is it's one simple step: Create a github release, push tag, walk away.  Will we now be back to editing `build.sbt` each time we release?

Reply:
I've checked other `typelevel` projects that use `sbt-ci-release` for releasing and also use MiMa — and it seems that every project does it manually. We release not so often, to be honest. So probably we get more benefits from checking binary compatibility. And yeah, for patch releases we can to not update that version as a whole. We should be checking the binary compatibility within the same major release.

Reply:
👍